src/regtech_mail_api/internal.py

```diff
@@ -8,12 +8,22 @@
 
 from regtech_mail_api.settings import EmailApiSettings
 from regtech_mail_api.models import Email
-from regtech_mail_api.mailer import create_mailer, get_header
+from regtech_mail_api.mailer import create_mailer
 
 settings = EmailApiSettings()
 
 router = Router()
 
+custom_months = {
+    "January": "Jan.",
+    "February": "Feb.",
+    "August": "Aug.",
+    "September": "Sept.",
+    "October": "Oct.",
+    "November": "Nov.",
+    "December": "Dec.",
+}
```

Comment:
only these need the abbreviations, and none of Mar - Jul?  If Mar - Jul should need abbreviations as well, I think `strftime` would work for all but May and September?

Reply:
Correct, according to Natalia the abbreviations expected by CFPB policy are:
![image](https://github.com/user-attachments/assets/1c7cfea2-ca9e-4b2e-a60e-ab21cb504c79)

Reply:
alrighty then, lol.